### PR TITLE
Nicer exception messages

### PR DESCRIPTION
**Pull request by connected-jallen**

### CODE REVIEW

File: rxandroidble/src/test/groovy/com/polidea/rxandroidble/RxBleClientTest.groovy [hunk 1]
```diff
@@ -518,4 +518,5 @@ class RxBleClientTest extends Specification {
         Thread.sleep(200) // Nasty :<
         true
     }
+
```
Comment (piotrek1543) [inline rxandroidble/src/test/groovy/com/polidea/rxandroidble/RxBleClientTest.groovy]: unnecessary formatting :-)

File: rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleServiceNotFoundException.java [hunk 2]
```diff
@@ -7,15 +7,11 @@
     private final UUID serviceUUID;
 
     public BleServiceNotFoundException(UUID serviceUUID) {
+        super("BLE Service not found with uuid " + serviceUUID);
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleServiceNotFoundException.java]: I think that `uuid` should be definitely with uppercase (I mean: `UUID` instead of `uuid`), as it is more frequent used version then with lowercase, you used is in another class  and  it relates to used there class name.

File: rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleScanException.java [hunk 3]
```diff
@@ -129,51 +131,46 @@ public Date getRetryDateSuggestion() {
...
+    private static String createMessage(int reason, Date retryDateSuggestion) {
+        return reasonDescription(reason) + " (code " + reason + ")" + retryDateSuggestionIfExists(retryDateSuggestion);
     }
 
-    private String reasonDescription() {
+    private static String reasonDescription(int reason) {
         switch (reason) {
             case BLUETOOTH_CANNOT_START:
-                return "BLUETOOTH_CANNOT_START";
+                return "Bluetooth cannot start";
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleScanException.java]: Really good piece of code. I like it :-)

File: rxandroidble/src/test/groovy/com/polidea/rxandroidble/RxBleClientTest.groovy (hunk 1 above)
Comment (uKL) [reply rxandroidble/src/test/groovy/com/polidea/rxandroidble/RxBleClientTest.groovy]: Correct, I'll update in another commit.

File: rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleServiceNotFoundException.java (hunk 2 above)
Comment (uKL) [reply rxandroidble/src/main/java/com/polidea/rxandroidble/exceptions/BleServiceNotFoundException.java]: Correct, I'll update in another commit.

**Comment by uKL** [comment]
Dear Contributor,

similar to many open source projects we kindly request to sign our CLA if you'd like to contribute to our project. This license is for your protection as a Contributor as well as the protection of Polidea; it does not change your rights to use your own Contributions for any other purpose.

You can find a link here: https://cla-assistant.io/Polidea/RxAndroidBle
